test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -34,13 +37,19 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, 1, Integer.valueOf(1));
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
+        assertIsAnyNull(null, 1, "a", null);
     }
 
+
     private void assertAreUnique(Object... objects) {
         assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
 
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertIsAnyNull(Object... objects) {
+      assertFalse(Utils.isAnyNull(Arrays.asList(objects)));
```

Comment:
Note that while your method is `assertIsAnyNull`, you are actually calling `assertFalse`, which should be giving you test case fails.

Do note that it is not necessary to abstract the `assertIsAnyNull` as a separate method (it was done previously to avoid having to type out Arrays.asList each time). However, it is also considered good practice to follow existing coding styles.